### PR TITLE
Config improvement

### PR DESCRIPTION
**Pull request by tuhuynh27**
- Fix config params name (h -> "host", p -> "port")
- Fix native transport loader for linux/mac non x86_64
- Add some code refactor and code comment

### CODE REVIEW

**Comment by tuhuynh27** [comment]
Why still fail lah?

**Comment by tuhuynh27** [comment]
Ok good now

File: core/src/main/java/dev/keva/core/config/KevaConfig.java [hunk 1]
```diff
@@ -16,12 +16,12 @@
...
 public class KevaConfig {
-    @ConfigProp(name = "hostname", defaultVal = "localhost")
-    @CliProp(name = "h", type = CliPropType.VAL)
+    @ConfigProp(name = "host", defaultVal = "localhost")
+    @CliProp(name = "host", type = CliPropType.VAL)
     private String hostname;
 
     @ConfigProp(name = "port", defaultVal = "6379")
-    @CliProp(name = "p", type = CliPropType.VAL)
+    @CliProp(name = "port", type = CliPropType.VAL)
```
Comment (axblueblader) [inline core/src/main/java/dev/keva/core/config/KevaConfig.java:24]: but why :cry:

Comment (tuhuynh27) [reply core/src/main/java/dev/keva/core/config/KevaConfig.java:24]: We will try to follow Redis Conf standard

Comment (tuhuynh27) [reply core/src/main/java/dev/keva/core/config/KevaConfig.java:24]: Check this https://redis.io/topics/config

Comment (tuhuynh27) [reply core/src/main/java/dev/keva/core/config/KevaConfig.java:24]:
```
./redis-server --port 6380 --replicaof 127.0.0.1 6379
```

They don't have `-p` option either

Comment (axblueblader) [reply core/src/main/java/dev/keva/core/config/KevaConfig.java:24]: ok i c

**Review by axblueblader — APPROVED** [review]

**Pull request merged**